llvm/lib/Target/DirectX/DXILLegalizePass.cpp

```diff
@@ -349,6 +396,7 @@ class DXILLegalizationPipeline {
     LegalizationPipeline.push_back(downcastI64toI32InsertExtractElements);
     LegalizationPipeline.push_back(legalizeFreeze);
     LegalizationPipeline.push_back(removeMemSet);
+    LegalizationPipeline.push_back(legalizeGetHighLowi64Bytes);
```

Comment:
This is going to be a more involved change because we already have a pass that replaces extract elements to handle the i64 indices case. I think we should add stages